raiden/tests/unit/test_upgrade.py

```diff
@@ -44,18 +43,29 @@ def test_upgrade_manager_restores_backup(tmp_path):
 
     storage = setup_storage(old_db_filename)
 
-    with patch('raiden.constants.RAIDEN_DB_VERSION', new=16):
+    with patch('raiden.utils.upgrades.RAIDEN_DB_VERSION', new=16):
         storage.update_version()
         storage.conn.close()
 
+    upgrade_functions = [Mock()]
+
+    upgrade_functions[0].return_value = 17
+
     web3, _ = create_fake_web3_for_block_hash(number_of_blocks=1)
-    with patch('raiden.utils.upgrades.older_db_file') as older_db_file:
-        older_db_file.return_value = str(old_db_filename)
+    with ExitStack() as stack:
```

Comment:
why not use pytest's [`monkeypatch.context`](https://docs.pytest.org/en/latest/reference.html#_pytest.monkeypatch.MonkeyPatch.context)? should be a bit simpler

Reply:
Done